.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
   # Avoid annoying focus problems when running tests
   # See discussion in e.g. https://github.com/spyder-ide/spyder/pull/6132
   - sudo apt-get -qq update
-  - sudo apt-get install -y matchbox-window-manager xterm libxkbcommon-x11-0
+  - sudo apt-get install -y matchbox-window-manager xterm libxkbcommon-x11-0 libspatialindex-dev
```

Comment:
Please remove `libspatialindex-dev` from here because we're going to use conda packages only.